rootfs-version-check/module/rootfs-version-check

```diff
@@ -22,6 +22,17 @@ fi
 
 case "$1" in
     Download)
+        if [ ! -x /usr/bin/mender-compare-versions ]; then
+            echo "Unable to locate mender-compare-versions executable."
+            exit 1
+        fi
+        # This script needs to be provided by the integrator and customized specifically for their naming scheme.
+        if ! /usr/bin/mender-compare-versions "$(cat header/artifact_name)" "$(cat current_artifact_name)"; then
```

Comment:
Where does `current_artifact_name` come from? I can not see that it is defined in the script?

Reply:
It's a file in the CWD provided by the update module framework.  Same as header/artifact_name.